Brief_Introduction_of_ggpubr.Rmd

```diff
@@ -0,0 +1,282 @@
+#Brief Introduction of ggpubr
+
+Hanlin Yan
+
+### 1. Introduction
```

Comment:
please remove the numbering in all headings/subheadings in the .Rmd file

Reply:
fixed